ArcGISRuntimeSDKQt_QMLSamples/Routing/RouteAroundBarriers/RouteAroundBarriers.qml

```diff
@@ -16,7 +16,6 @@
 
 import QtQuick 2.6
 import QtQuick.Controls 2.2
-//import Esri.ArcGISExtras 1.1
```

Comment:
is extras not needed? If so, please remove this all together

Reply:
I have removed it

Reply:
of course, sorry about that. Need more ☕

Reply:
no problem!